orientdb/src/test/java/org/hibernate/datastore/ogm/orientdb/OrientDBSimpleTest.java

```diff
@@ -0,0 +1,280 @@
+		try {
+			em.getTransaction().begin();
+			Customer customer = em.find( Customer.class, Long.valueOf( 2L ) );
+			em.refresh( customer );
+			LOG.log( Level.INFO, "read entity properties:" );
+			LOG.log( Level.INFO, "customer.getbKey():{0}", customer.getbKey() );
+			LOG.log( Level.INFO, "customer.getName(): {0}", customer.getName() );
+			LOG.log( Level.INFO, "customer.getRid(): {0}", customer.getRid() );
+			assertEquals( Long.valueOf( 2L ), customer.getbKey() );
+			assertNotNull( customer.getRid() );
+		}
+		finally {
+			em.getTransaction().commit();
+		}
+	}
+
+	@Test
+	public void test2FindPizza() {
+		LOG.log( Level.INFO, "start" );
+		try {
+			em.getTransaction().begin();
+			Pizza pizza = em.find( Pizza.class, Long.valueOf( 2L ) );
+			em.refresh( pizza );
+			LOG.log( Level.INFO, "read entity properties:" );
+			LOG.log( Level.INFO, "pizza.getBKey():{0}", pizza.getbKey() );
+			LOG.log( Level.INFO, "pizza.getName(): {0}", pizza.getName() );
+			LOG.log( Level.INFO, "pizza.getbKey(): {0}", pizza.getbKey() );
+			assertEquals( Long.valueOf( 2L ), pizza.getbKey() );
+			assertNotNull( pizza.getRid() );
+		}
+		finally {
+			em.getTransaction().commit();
+		}
+
+	}
+
+	@Test
+	public void test3CreateNativeQuery() {
+		LOG.log( Level.INFO, "start" );
+		try {
+			em.getTransaction().begin();
+			LOG.log( Level.INFO, "query: select from Customer" );
+			Query query = em.createNativeQuery( "select from Customer", Customer.class );
+			List<Customer> customers = query.getResultList();
+			assertFalse( "Customers must be", customers.isEmpty() );
+			assertEquals( Long.valueOf( 2L ), customers.get( 0 ).getbKey() );
+
+			LOG.log( Level.INFO, "query: select from {0}", customers.get( 0 ).getRid().toString() );
+			query = em.createNativeQuery( "select from " + customers.get( 0 ).getRid().toString(), Customer.class );
+			customers = query.getResultList();
+			assertFalse( "Customers must be", customers.isEmpty() );
+
+			LOG.log( Level.INFO, "query: select from Customer where name=:name" );
+			query = em.createNativeQuery( "select from Customer where name=:name", Customer.class );
+			query.setParameter( "name", "test" );
+			customers = query.getResultList();
+			assertFalse( "Customers must be", customers.isEmpty() );
+
+			LOG.log( Level.INFO, "query: select from Customer where name='test'" );
+			query = em.createNativeQuery( "select from Customer where name='test'", Customer.class );
+			customers = query.getResultList();
+			assertFalse( "Customers must be", customers.isEmpty() );
+
+		}
+		finally {
+			em.getTransaction().commit();
+		}
+
+	}
+
+	@Test
+	public void test4UpdateCustomer() {
+		LOG.log( Level.INFO, "start" );
+		try {
+			em.getTransaction().begin();
+			Long id = Long.valueOf( 2L );
+			Customer customer = em.find( Customer.class, id );
+			customer.setName( "Ivahoe" );
+			int oldVersion = customer.getVersion();
+			LOG.log( Level.INFO, "old version:{0}", oldVersion );
+			em.merge( customer );
```

Comment:
You don't need to use merge in this case.

Just for reference, the em.merge() has a returned value not sure if it works as expected otherwise.